src/main/java/reposense/git/GitUtil.java

```diff
@@ -79,15 +89,51 @@ static String convertToGitFormatsArgs(List<Format> formats) {
 
     /**
      * Returns the {@code String} command to specify the globs to exclude for `git log` command.
+     * Also checks that every file in {@code ignoreGlobList} is within the given repository {@code root}
```

Comment:
I think `checks that every glob in {@code ignoreGlobList} only targets files within the given repository {@code root}` is more correct?